### ISSUE TITLE
'System.MissingMethodException' in version 0.99.67

### ISSUE THREAD

**Issue by Rotvig** [rebus-org/Rebus#482]
I was using Rebus 2 version 0.99.63 and upgraded to version 0.99.67 and then i got this error:

```
Exception thrown: 'System.MissingMethodException' in Rebus.dll

Additional information: Method not found: 'Rebus.Transport.Msmq.MsmqTransportConfigurationBuilder Rebus.Transport.Msmq.MsmqTransportConfigurationExtensions.UseMsmq(Rebus.Config.StandardConfigurer`1<Rebus.Transport.ITransport>, System.String)'.
```

I tried to localise the error so i removed all my settings and got the following code where the error is thrown. It seems like when im calling ".Transport(x => x.UseMsmq("random"))" it fails.
```c#
var activator = new BuiltinHandlerActivator();
 Configure.With(activator)
      .Transport(x => x.UseMsmq("random"))
      .Start();

return activator.Bus;
```

Its all working fine in version 0.99.63.

**Comment by mookid8000** [comment]
That's weird! 😐 I suspect it is caused by your Rebus.dll reference used during compilation is not the Rebus.dll your application ends up running with. This can happen if you use R# ALT+ENTER to add references to NuGet-imported DLLs from other projects instead of adding them as proper NuGet references.

The reason I have this suspicion is because Rebus had its `UseMsmq` extension method changed slightly in [0.99.65](https://github.com/rebus-org/Rebus/blob/master/CHANGELOG.md#09965) changing the `void` configuration method into one returning a builder, thus enabling you to customize MSMQ queues created by Rebus.

This change allows e.g. for setting default permissions on newly created queues to `FullControl` for `Everyone` like this:

    Configure.With(activator)
        .Transport(t =>
        {
            t.UseMsmq("msmq-config-builder")
                .OnCreated(queue =>
                {
                    var user = new SecurityIdentifier(WellKnownSidType.WorldSid, null)
                                .Translate(typeof(NTAccount))
                                .ToString();

                    queue.SetPermissions(user, MessageQueueAccessRights.FullControl);
                });
        })
        .Start();

(which should obviously only be done in environments where you do not worry that much about these things 😄 )

In summary: I think you should ensure that projects referencing Rebus have it imported via NuGet and are all up-to-date. Please let me know if that does not fix it 😄

**Comment by Rotvig** [comment]
Your suspicion was correct. It fixed it and its all running now. Thanks for the help :)

**Comment by mookid8000** [comment]
you're welcome 😄
